core/bin/external_node/src/main.rs

```diff
@@ -567,6 +540,53 @@ async fn main() -> anyhow::Result<()> {
     )
     .await?;
 
+    // Revert the storage if needed.
+    let reverter = BlockReverter::new(
+        NodeRole::External,
+        config.required.state_cache_path.clone(),
+        config.required.merkle_tree_path.clone(),
+        None,
+        connection_pool.clone(),
+        L1ExecutedBatchesRevert::Allowed,
+    );
+
+    let mut reorg_detector = ReorgDetector::new(main_node_client.clone(), connection_pool.clone());
+    // We're checking for the reorg in the beginning because we expect
```

Comment:
Micronit: the spacing feels a bit odd here with comments being shorter even than surrounding code lines, usually (though not consistently) we align comments to 120.

Reply:
done